saleor/graphql/discount/dataloaders.py

```diff
@@ -227,3 +227,7 @@ def batch_load(self, keys):
         for discount in discounts:
             discount_map[discount.order_id].append(discount)
         return [discount_map.get(order_id, []) for order_id in keys]
+
+
+def load_request_discounts(request):
```

Comment:
Could be just `load_discounts` I guess? Seems consistent with other "load_*" functions.